src/test/java/com/productdock/rbc2024/mapper/bookmapper/BookMapperShould.java

```diff
@@ -1,32 +1,49 @@
 package com.productdock.rbc2024.mapper.bookmapper;
 
+import com.productdock.rbc2024.dto.BookDetailsDto;
+import com.productdock.rbc2024.dto.EditBookDetailsDto;
 import com.productdock.rbc2024.mapper.BookMapper;
+import com.productdock.rbc2024.model.Book;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class BookMapperShould {
 
     private BookMapper bookMapper;
 
-
     @BeforeEach
     void setUp() {
         bookMapper = new BookMapper();
     }
 
     @Test
     void convertModelToBookDetailsDto() {
-        // given
-        var book = BookMapperSetUp.createBook();
+        Book bookToConvert = BookMapperSetUp.createBook();
```

Comment:
Please, use given, when and then sections. You don't have to use comments for each section, but you can separate each section with a blank line. The tests would be more readable.